scripts/routing.js

```diff
@@ -128,34 +135,42 @@ async function calculateFullRoute(fromCoordinates, toCoordinates) {
 	let coords1 = ol.proj.fromLonLat([routeSummaryBike.bbox[0], routeSummaryBike.bbox[1]]);
 	let coords2 = ol.proj.fromLonLat([routeSummaryBike.bbox[2], routeSummaryBike.bbox[3]]);
 	let convertedBbox = [coords1[0], coords1[1], coords2[0], coords2[1]];
-	let viewableBox = [map.getSize()[0], map.getSize()[1] - document.getElementById("placeSearchMenu").clientHeight];
+	let menuHeight;
+	if (document.getElementById("placeSearchMenu")) menuHeight = document.getElementById("placeSearchMenu").clientHeight;
+	else if (document.getElementById("stationMenu"))
+		menuHeight = document.getElementById("stationMenu").clientHeight + 30;
+	else menuHeight = 0;
+	let viewableBox = [map.getSize()[0], map.getSize()[1] - menuHeight];
 	map.getView().fit(convertedBbox, {
 		size: viewableBox,
-		padding: [50, 100, document.getElementById("placeSearchMenu").clientHeight + 50, 100],
+		padding: [50, 100, menuHeight + 50, 100],
 		maxZoom: 18,
 	});
 
 	// Show the start navigation button and route details panel
-	if (document.querySelector("#placeSearchMenu")) {
-		let placeSearchMenuElement = document.querySelector("#placeSearchMenu");
-
-		let startNavigationButtonElement = document.createElement("div");
-		startNavigationButtonElement.id = "startNavigationButton";
-		startNavigationButtonElement.innerHTML = '<i class="bi bi-sign-turn-slight-right"></i>';
-
-		startNavigationButtonElement.addEventListener("click", () => startNavigation(walkingOnly));
-
-		let routeDetailsElement = document.createElement("div");
-		routeDetailsElement.id = "routeDetails";
-		routeDetailsElement.innerHTML = `<i class="bi bi-clock"></i>&nbsp;${new Date(totalTime * 1000)
-			.toISOString()
-			.slice(11, 19)}&nbsp;&nbsp;&nbsp;<i class="bi bi-signpost"></i>&nbsp;${
-			Math.round((totalDistance / 1000) * 10) / 10
-		}km`; // round distance to 1 decimal place
-
-		placeSearchMenuElement.appendChild(startNavigationButtonElement);
-		placeSearchMenuElement.appendChild(routeDetailsElement);
-	}
+	let targetElement = null; // either placeSearchMenu or stationMenu
+
+	// check which element we need to append to
+	if (document.querySelector("#placeSearchMenu")) targetElement = document.querySelector("#placeSearchMenu");
+	else if (document.querySelector("#stationMenu")) targetElement = document.querySelector("#stationMenu");
+
+	// create start Navigation button
+	let startNavigationButtonElement = document.createElement("div");
```

Comment:
tenta usar mais const pff
```suggestion
	const startNavigationButtonElement = document.createElement("div");
```